app/src/main/res/values-de/strings.xml

```diff
@@ -50,4 +50,7 @@
     <string name="export_as_pdf">Als PDF exportieren!</string>
     <string name="export_your_scans_and_edits_in_pdf_format">Exportieren Sie Ihre Scans im PDF-Format.</string>
     <string name="finish">Fertig</string>
+    <string name="okay">Okay</string>
+    <string name="camera_permission_info">Die Anwendung benötigt Zugriff auf die Kamera, um ein Foto aufzunehmen und zuzuschneiden, aus dem Text extrahiert wird.\n\nBitte denken Sie daran, wenn die Berechtigung zweimal verweigert wird, müssen Sie sie manuell in den Einstellungen ändern.</string>
```

Comment:
Text should be:
Die Anwendung benötigt Zugriff auf die Kamera, um ein Foto aufzunehmen und zuzuschneiden und aus diesem Foto dem Text extrahieren.\n\nBitte denken Sie daran, wenn die Berechtigung zweimal verweigert wird, müssen Sie sie manuell in den Einstellungen ändern.